Extension/src/LanguageServer/configurations.ts

```diff
@@ -699,13 +699,13 @@ export class CppProperties {
                 for (let pathArray of [ (this.CurrentConfiguration.browse ? this.CurrentConfiguration.browse.path : undefined),
                         this.CurrentConfiguration.includePath, this.CurrentConfiguration.macFrameworkPath, this.CurrentConfiguration.forcedInclude ] ) {
                     if (pathArray) {
-                        for (let path of pathArray) {
-                            paths.add(path);
+                        for (let curPath of pathArray) {
+                            paths.add('"' + curPath + '"');
                         }
                     }
                 }
                 if (this.CurrentConfiguration.compileCommands) {
-                    paths.add(this.CurrentConfiguration.compileCommands);
+                    paths.add('"' + this.CurrentConfiguration.compileCommands + '"');
```

Comment:
This will work. If you feel like it you can also use a format string which is a little bit less of a hassle to modify. https://basarat.gitbooks.io/typescript/docs/template-strings.html

Reply:
Yeah, I like that style better -- I forgot TypeScript had that.